tests/components/samsungtv/test_media_player.py

```diff
@@ -511,12 +511,14 @@ def mock_now():
 
 async def test_turn_on_with_turnon(hass, remote):
     """Test turn on."""
-    await setup_samsungtv(hass, MOCK_CONFIG)
-    assert await hass.services.async_call(
-        DOMAIN, SERVICE_TURN_ON, {ATTR_ENTITY_ID: ENTITY_ID}, True
-    )
-    # key and update called
-    assert False  # TODO
+    with patch(
```

Comment:
Make sure to use `asynctest.patch` to patch coroutines. It works for both normal functions and coroutines so is backwards compatible.